graders/python/Dockerfile

```diff
@@ -10,6 +10,6 @@ RUN yum -y update \
     && yum install -y python36u python36u-devel python36u-pip \
     && yum install -y graphviz graphviz-devel \
     && ln -s /usr/bin/python3.6 /usr/bin/python3 \
-    && python3 -m pip install --no-cache-dir -r /requirements.txt
+     ; python3 -m pip install --no-cache-dir -r /requirements.txt
```

Comment:
What's the reason for this change?

Reply:
When I was building the docker image locally on my machine it was error-ing out because python3 already existed, so I made it fail silently if it can't create the symlink

Reply:
Good point -- I think the right fix here is to apply the same `python3` update as @trombonekenny did for `plbase`: https://github.com/PrairieLearn/PrairieLearn/pull/1719